mysql-native/src/main/java/org/ballerinalang/mysql/utils/ModuleUtils.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.mysql.utils;
+
+import io.ballerina.runtime.api.Environment;
+import io.ballerina.runtime.api.Module;
+
+/**
+ * Utility functions relevant to module operations.
+ *
+ * @since 2.0.0
```

Comment:
Is this version correct?

Reply:
Corrected in https://github.com/ballerina-platform/module-ballerinax-mysql/pull/104/commits/58551087f8f7b7517137fa8d16da9a7a24ae710c